data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+		system destination
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis entering harvests plunders
+			confusion 20
+		system destination
+		fleet "Small Northern Pirates"
+	npc
+		government Pirate
+		personality nemesis staying harvests plunders
+			confusion 20
+		system destination
+		fleet "Small Northern Pirates" 2
+	
+	npc accompany save
+		government "Merchant"
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Freighter" 3
+	on complete
+		payment 100000
+		dialog "The captain of the <npc> thanks you for escorting their ships safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+
+mission "Food Convoy [1]"
+	name `Food convoy to <planet>`
+	job
+	repeat
+	description `The captain of the <npc> will pay you <payment> to escort his convoy of ships carrying food to <destination> by <date>.`
+	to offer
+		random < 25
+		"combat rating" > 60
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 9
+		attributes "near earth" "paradise"
+	npc
+		government "Pirate"
+		personality nemesis harvests plunders
+		system destination
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis entering harvests plunders
+		system destination
+		fleet "Small Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Large Northern Pirates"
+	
+	npc accompany save
+		government "Merchant"
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Behemoth"
+				"Freighter" 2
+	on complete
+		payment 160000
+		dialog "The captain of the <npc> thanks you for escorting their ships safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+
+mission "Food Convoy [2]"
+	name `Food convoy to <planet>`
+	job
+	repeat
+	description `The captain of the <npc> will pay you <payment> to escort his large convoy of ships carrying food to <destination> by <date>.`
+	to offer
+		random < 20
+		"combat rating" > 90
+	source
+		attributes "dirt belt"
+	destination
+		distance 4 10
+		attributes "near earth" "paradise"
+	npc
+		government "Pirate"
+		personality nemesis harvests plunders
+		system destination
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis entering harvests plunders
+		system destination
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+	
+	npc accompany save
+		government "Merchant"
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Behemoth" 2
+	on complete
+		payment 220000
```

Comment:
That's a lot of money, even for large pirate fleets.

Reply:
The distance that needs to be traveled can also be over double that of normal escort jobs, but if you want me to toughen up the pirates I can.